### PR TITLE
Feat/nowifi

### PR DESCRIPTION
**Pull request by samuellye98**
Allow app to load last verified even without wifi

### CODE REVIEW

**Comment by yehjxraymond** [comment]
Expo QR
Published to https://exp.host/@workpasssg/sg-workpass?release-channel=29

**Comment by yehjxraymond** [comment]
Expo QR
Published to https://exp.host/@workpasssg/sg-workpass?release-channel=29

**Comment by samuellye98** [comment]
https://github.com/sgworkpass/mobile/issues/28

**Comment by yehjxraymond** [comment]
Expo QR
Published to https://exp.host/@workpasssg/sg-workpass?release-channel=29

File: src/components/Camera/QRScanner.tsx [hunk 1]
```diff
@@ -57,15 +61,20 @@ class QRScanner extends React.Component<QRScannerProps> {
       "Do you want to overwrite your current profile?",
       [
         {
-          text: "No"
+          text: "No",
+          onPress: () => {
+            this.setState({ isProcessingQr: false });
```
Comment (yykcool) [inline src/components/Camera/QRScanner.tsx]: onPress: () => this.setState({ isProcessingQr: false });

**Comment by yehjxraymond** [comment]
Expo QR
Published to https://exp.host/@workpasssg/sg-workpass?release-channel=29

**Pull request merged**